broker/README.md

```diff
@@ -42,7 +42,8 @@ install KEDA as documented [here](https://keda.sh/docs/latest/deploy/).
 install the broker-controller from this repository:
 
 ```
-ko apply -f config/broker/
+kubectl create namespace knative-eventing
+kubectl apply -f config/broker/
```

Comment:
This is actually not going to work with just plain kubectl. You have to use ko because the images are using ko:// so unless you use ko they won't get built and the controller won't start, well, nothing will work :)